Sources/Basics/AuthorizationProvider.swift

```diff
@@ -96,9 +109,16 @@ public struct NetrcAuthorizationProvider: AuthorizationProvider {
             callback(.failure(AuthorizationProviderError.other("Failed to update netrc file at \(self.path): \(error)")))
         }
     }
+    
+    public func remove(for url: URL, callback: @escaping (Result<Void, Error>) -> Void) {
+        callback(.failure(AuthorizationProviderError.other("User must edit netrc file at \(self.path) manually to remove entries")))
```

Comment:
We can't reliably update `.netrc` (e.g., we might delete comments by accident), so simply not support it for now.